src/components/Menu/MenuItem/MenuItem.scss

```diff
@@ -0,0 +1,267 @@
+.menu-item:focus {
+  outline: none;
+}
+
+// MOBILE
+@media screen and (max-width: 1343px) {
+  $menu-item__first-position-left: 28px;
+
+  @for $i from 1 through 6 {
+    .menu-item#{$i} { left: $menu-item__first-position-left + (($i - 1) * 58px); }
+  }
+
+  .menu-item {
+    background-color: transparent;
+    border: none;
+    border-radius: 18px;
+    padding: 0;
+    position: absolute;
+    width: 36px;
+    transition: left 0s;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    height: 64px;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  }
+  .menu-item:active {
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: 20;
+    transition: left 2s 0.25s;
+
+
+    @media (prefers-color-scheme: light) {
+      .menu-item__tooltip {
+        background-color: var(--primaryColor);
+      }
+  
+    }
+    @media (prefers-color-scheme: dark) {
+      .menu-item__tooltip {
+        background-color: #FFFFFF;
+      }
+    }
+  }
+  // .menu-item:hover > .menu-item__icon {
+  //   animation: spinning-icon 1.75s linear forwards;
+  // }
+  .menu-item:active > .menu-item__tooltip {
+    height: 100%;
+    width: 100%;
+    transition: background-color 0.5s 0.25s ease, width 0.5s 0.25s ease, height 0.5s 0.25s ease;
+  }
+  .menu-item:active > .menu-item__icon {
+    left: $menu-item__first-position-left;
+    transition: left 2s 0.5s, visibility 0s 0.5s;
+  }
+  .menu-item:active > .tooltip__text {
+    margin-right: 48px;
+  }
+  .menu-item--active:active > .menu-item__icon--end {
+    visibility: visible;
+  }
+  .menu-item--active:active > .menu-item__icon--start {
+    visibility: hidden;
+  }
+
+  .menu-item__tooltip {
+    box-sizing: border-box;
+    display: inline-flex;
+    align-items: center;
+    font-family: 'Raleway', serif;
```

Comment:
The `font-family` should be defined globally and not for each element, except it really uses another font

Reply:
Done ✅